taipy/gui/_renderers/factory.py

```diff
@@ -618,6 +619,19 @@ class _Factory:
             ]
         )
         ._set_propagate(),
+        "notification": lambda gui, control_type, attrs: _Builder(
+            gui=gui,
+            control_type=control_type,
+            element_name="Notification",  
+            attributes=attrs,
+        )
+        .set_value_and_default()  
+        .set_attributes(
+            [
+                ("severity", PropertyType.string),  # severity is a simple string property
+                ("message", PropertyType.dynamic_string),  # message is now a dynamic string
```

Comment:
message shouldn't be described here
you need to pass var_type= PropertyType.dynamic_string to set_value_and_default() line 628